frb_rust/src/third_party/wasm_bindgen/worker_pool.rs

```diff
@@ -42,6 +59,16 @@ impl WorkerPool {
         )
     }
 
+    pub fn new_with_max_workers(max_workers: usize) -> Result<WorkerPool, JsValue> {
```

Comment:
nit: wondering whether we should move the "new_with_config" thing here, i.e. remove

* new
* new_with_max_workers
* new_raw

and have only one single new_with_config that accepts a config

b/c otherwise we have a lot of constructors here...

For backward compatibility we may have to keep the new and new_raw there, but they can be deprecated